test/unit/host_test.rb

```diff
@@ -1907,7 +1907,32 @@ def to_managed!
     assert host.errors[:provision_method].include?("can't be updated after host is provisioned")
   end
 
-  test "#image_build? must be true when provision_method is image" do
+  test "#provision_methods must include build and image by default" do
+    host = FactoryGirl.create(:host, :managed)
+    assert host.provision_methods.key?('build')
```

Comment:
`assert_include 'build', host.provision_methods.keys` helps as it'll print out the contents of the array if it fails